docs/book/src/faq.md

```diff
@@ -92,6 +92,15 @@ securityContext:
 ```
 However, note that this problem is fixed and will not occur if you deploy the project in high versions (maybe >= 1.22).
 
+## Kubebuilder  Make install fails with this error: Too long: must have at most 262144 bytes
+
+Kubernetes api has a size limit and if the CRD is too big with so many long descriptions it will cause the failure.
```

Comment:
```suggestion
The Kubernetes API has a size limit. Then, when the apply is used the API will add the annotation last-applied-configuration with all previous configuration which is exceeding this amount. [More info](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/#how-to-create-objects)

Therefore, you have a few options to workround this scenario such as:

- You can use the option `maxDescLen=0` to remove the description of your CRDs. Following an example:

```shell

 .PHONY: manifests 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects. 
     # Note that the option maxDescLen=0 was added in the default scaffold in order to sort out the issue 
     # Too long: must have at most 262144 bytes. By using kubectl apply to create / update resources an annotation 
     # is created by K8s API to store the latest version of the resource ( kubectl.kubernetes.io/last-applied-configuration). 
     # However, it has a size limit and if the CRD is too big with so many long descriptions as this one it will cause the failure. 
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases 
```

- You can review the design of your APIs and see if it has not more specs than should be by hurting single responsibility principle for example. So that you might to re-design them. 

Note that the ideal solution would be to use client-side apply. However, it still not supported by Controller-gen and consequently by kubebuilder. For further information see: https://github.com/kubernetes-sigs/controller-tools/pull/536

```

Can you shape the text, but it would be something like ^ 

Also, we need to add a note in the tutorial to explain the problem and that the example of the tutorial is using maxDescLen=0. It can be in a follow up PR and you can link this faq section there.